src/main/java/at/hannibal2/skyhanni/config/features/GardenConfig.java

```diff
@@ -799,6 +799,16 @@ public static class EliteFarmingWeightConfig {
         @ConfigOption(name = "Show below 200", desc = "Show the farming weight data even if you are below 200 weight.")
         @ConfigEditorBoolean
         public boolean ignoreLow = false;
+
+        @Expose
+        @ConfigOption(name = "Show outside garden", desc = "Show the farming weight outside of the garden.")
+        @ConfigEditorBoolean
+        public boolean showOutsideGarden = false;
+
+        @Expose
+        @ConfigOption(name = "Show outside SB", desc = "Show the farming weight outside of SkyBlock.")
```

Comment:
Why? Im sure we don't want this, also currently all the data gets reset on world switch anyway.

Reply:
I want it
Also the data didn't get reset on world switch? Leaderboard just didn't update outside of sb before.
I've fixed that now though. It'll use the selected profile from the elite api call (no new api call) when outside of sb.